README.md

```diff
@@ -215,7 +215,17 @@ Same using pronto, fetching ontology from obolibrary
 runoak -i obolibrary:go.obo  viz GO:0005773
 ```
 
-## Potential Refactoring
+## Developer notes
+### Local project setup
+Prerequisites:
+1. Python 3.10+
+2. [Poetry](https://python-poetry.org/)
+
+Setup steps:
+1. Clone this repository.
```

Comment:
would be better to explicitly write the commands in a shell-like format, including the git clone command, the cd, etc

See https://github.com/biopragmatics/bioregistry#-installation for an example

Reply:
Not a bad suggestion. Done~

I did leave off the `$` though. Even though it's a common convention, I have some beef with that: (i) makes copy/paste more tedious, (ii) the code block formatting already helps with indication, (iii) not every prompt uses `$`.